doc/classes/@GlobalScope.xml

```diff
@@ -494,22 +495,22 @@
 			<return type="bool" />
 			<param index="0" name="instance" type="Variant" />
 			<description>
-				Returns whether [param instance] is a valid object (e.g. has not been deleted from memory).
+				Returns [code]true[/code] if [param instance] is a valid Object (e.g. has not been deleted from memory).
```

Comment:
```suggestion
				Returns [code]true[/code] if [param instance] is a valid [Object] (e.g. has not been deleted from memory).
```

Curious:

```gdscript
print(is_instance_valid(123)) # false
print(is_instance_valid("abc")) # false
print(is_instance_valid(Color.RED)) # false
```

Didn't think about it before.

Reply:
The description stands corrected. It could be more explicit, but this PR is too large as is.